test/jdk/java/net/URLConnection/RequestProperties.java

```diff
@@ -79,4 +82,29 @@ private static boolean hasFtp() {
             return false;
         }
     }
+
+    /**
+     * Test that {@link URLConnection#setRequestProperty(String, String)} throws
+     * an {@link IllegalStateException} when already connected
+     *
+     * @throws Exception
```

Comment:
This @throws seems superfluous. Please remove.

Reply:
Removed.